lifecycle-operator/apis/lifecycle/v1alpha1/keptnappversion_conversion_test.go

```diff
@@ -51,18 +51,6 @@ func TestKeptnAppVersion_ConvertFrom(t *testing.T) {
 								Version: "4.5.6",
 							},
 						},
-						PreDeploymentTasks: []string{
```

Comment:
why did you remove this part ? KeptnAppVersion should stay the same

Reply:
`PreDeploymentTasks` was removed in `v1beta1.KeptnAppSpec` as such leaving this throws an error that such field doesn't exist

Reply:
but these fields should remain in KeptnAppVersion

Reply:
That's right as part of `DeploymentTaskSpec` no longer `KeptnAppSpec`